storage/vc/CMakeLists.txt

```diff
@@ -9,6 +9,6 @@ algebra_add_library( algebra_vc_storage vc_storage
    "include/algebra/storage/vc.hpp"
    "include/algebra/storage/impl/vc_array4_wrapper.hpp" )
 target_link_libraries( algebra_vc_storage
-   INTERFACE algebra::common Vc::Vc )
+   INTERFACE algebra::common algebra::common_storage Vc::Vc )
```

Comment:
:confused: I don't see any of the code in `algebra::vc_storage` changing in this PR in the end. I believe the change in this line should be undone. :thinking: